travel_platform/travel_app/views.py

```diff
@@ -3,10 +3,27 @@
 
 from django.http import Http404
 
+from django.http import JsonResponse
+
 from django.shortcuts import render
 
 from .models import Query
 
 
 def index(request):
     return render(request, 'travel_app/main.html')
+
+
+def coord(request):
+    """
+    View added for checking of interaction of map from landing page with 
+    server.
+    :param request: 
+    :return: 
+    """
+
+    """TODO: finish with interaction with API"""
```

Comment:
For `NOTE`, `TODO`, `FIXME` please use this format:
`# TODO pivanchy: Refactor view during API integration.`

Where `pivanchy` - author, who is responsible for this change.
`Refactor view during API integration` - your custom message, which describes action to do.